torch/_inductor/scheduler.py

```diff
@@ -2111,6 +2111,9 @@ def score_fusion(self, node1: BaseSchedulerNode, node2: BaseSchedulerNode):
         )
         return (
             node1.is_template() == config.epilogue_fusion_first and memory_score > 0,
+            # give reduction followed by pointwise less priority since the saving is small
+            # while the data access pattern for pointwise may be worse
+            not (node1.is_reduction() and node2.is_pointwise()) and memory_score > 0,
```

Comment:
The next (pre-existing) line already discourages this pattern (reduction + non-reduction in any order). Do we need a more specific match in front?